openedx/stanford/cms/envs/test.py

```diff
@@ -1,7 +1,6 @@
 from cms.envs.test import *
 
 
-FEATURES['ALLOW_COURSE_RERUNS'] = True
```

Comment:
We no longer need to re-override for Stanford tests,
since that's now the default again.